doc/topics/releases/fluorine.rst

```diff
@@ -180,6 +180,15 @@ The syntax is as follows:
     - :py:mod:`yumpkg <salt.modules.yumpkg>`
     - :py:mod:`zypper <salt.modules.zypper>`
 
+Wheel Peering Options
+=====================
+
+A ``publish.wheel`` module has been added, allowing access to the wheel
+functions on the master.
```

Comment:
Again, I think we need to be extremely explicit about what this means from a security perspective. I am already extremely wary of this feature and I want us to go way overboard on making sure people understand what turning this on would do.

Reply:
I added a warning to this.